unit_tests/sources/declarative/decoders/test_json_decoder.py

```diff
@@ -54,7 +54,7 @@ def test_jsonl_decoder(requests_mock, response_body, expected_json):
 def large_event_response_fixture():
     data = {"email": "email1@example.com"}
     jsonl_string = f"{json.dumps(data)}\n"
-    lines_in_response = 2  # ≈ 58 MB of response
+    lines_in_response = 2_000_000  # ≈ 58 MB of response
```

Comment:
i might've accidentally changed this by accident in a past review

Reply:
Ha I remember that PR, too.